.ember-cli

```diff
@@ -1,4 +1,4 @@
 {
-  "port": 4400,
+  "port": 4200,
```

Comment:
This was 4400 so it could peacefully coexist locally with preprints at 4200. Is that not a problem with this update?

Reply:
No, we have to override it in the OSF anyway. So for simplicity, leaving it as 4200 is better. I'll  have another PR that updates the osf.io docker-compose.yml file with the correct port numbers for each ember app. You can also use `yarn start --port 4400` if you are running it locally outside docker, or `docker run -it <image> yarn start --port 4400` inside docker.

Reply:
Makes sense 👍